### DISCUSSION TITLE
Wide layout not working

### DISCUSSION THREAD

**Discussion by jlconlin** [mmistakes/minimal-mistakes#4586] [Q&A]
I'm trying to make the page wide using `classes: wide` in my frontmatter, but I'm not having any luck. I'm using version 4.24 so that should be sufficient. I've tried using `classes: wide` in the defaults as well, but to no avail. 

I'm not even sure where to start looking why this isn't working. Please advise.

I have created my own layout `post` that "inherits" from `single` and adds one more item. This is the only thing I can think of that may be interfering with the wide layout.

```markdown
---
layout: single
---

{{ content }}

{% include mentions-list.html %}
```

**Comment by jlconlin** [comment]
Bumping this up in the hopes that I get a response.

**Reply by ryan-p-randall** [reply]
I'm on the remote theme, which is also version 4.24.0. I just tried a local post and was able to successfully use the wide layout. 

So without seeing your files, I'd guess it's one of your layout modifications—or perhaps that the YAML in your post is conflicting with any defaults you have set in your overall config file.

For whatever it's worth, I have `layout: single` set in my site's config file. Then I added `classes: wide` as one line of the YAML of the test post that worked successfully for me.

I hope that helps! If it doesn't, perhaps you'll feel comfortable sharing a link to the files you're using?

**Reply by jlconlin** [reply]
I removed all my defaults thinking they were getting in the way. The only defaults I now have is `layout: single`. I used `layout: single` and `classes: wide` in the YAML frontmatter for a page under `_pages`. Looks like this is going to require more digging.

I'm sorry I can't point to a file you can see. I'm prohibited by my employer.
